src/App.js

```diff
@@ -129,6 +129,26 @@ class App extends React.Component {
     this.setState({ notes: [...notes, newTopic] });
   };
 
+  deleteNote = (selectedNote) => {
+    const { notes } = this.state;
+    const newNotes = notes.length > 0 && _.map(notes, (topic, topicIdx) => {
+      if (topicIdx !== selectedNote.topicIndex) {
+        return { ...topic };
+      }
+      else {
+        const afterDeletion = _.filter(topic.notes, (id) => {
+          return selectedNote.noteId !== id.id;
+        })
+        return {
+          topic: topic.topic,
+          notes: afterDeletion,
+        };
+      }
+    });
+    this.setState({ notes: newNotes});
```

Comment:
I have a save notes method you can use above so that it updates the file  system as well as the state